### PR TITLE
chore(Redesign Servizi): [IARS-40] Removes the currentSelectedService reducer and action

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR removes the currentSelectedService store section

Updated configure and persistor rule to remove persisted information

### CODE REVIEW

**Review by pietro909 — APPROVED** [review]

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: ts/boot/configureStoreAndPersistor.ts [hunk 1]
```diff
@@ -242,6 +242,25 @@ const migrations: MigrationManifest = {
...
+        services: {
+          servicePreference: entities.services.servicePreference,
+          byId: entities.services.byId,
+          byOrgFiscalCode: entities.services.byOrgFiscalCode,
+          visible: entities.services.visible,
+          readState: entities.services.readState,
+          firstLoading: entities.services.firstLoading
+        }
+      }
+    };
```
[the remark this answers is not in the document]
Comment (CrisTofani) [reply ts/boot/configureStoreAndPersistor.ts:263]: the first point: it was a CP&Paste error, my bad! of course I mean entities 🤕

[the remark this answers is not in the document]
Comment (CrisTofani) [reply ts/boot/configureStoreAndPersistor.ts:263]: the second point I'm going to fix the rule

File: ts/boot/configureStoreAndPersistor.ts [hunk 2]
```diff
@@ -242,6 +242,25 @@ const migrations: MigrationManifest = {
       ...state,
       content: { ...content, idps: remoteUndefined }
     };
+  },
+  // Version 17
+  // we removed currentSelectedServices from entities.services
+  "17": (state: PersistedState) => {
```
Comment (Undermaken) [inline ts/boot/configureStoreAndPersistor.ts:248]: you don't need this migration anymore.
It has not effect and producer an error (`entities` here is undefined)

**Review by Undermaken — CHANGES_REQUESTED** [review]

**Review by Undermaken — APPROVED** [review]

**Comment by Undermaken** [comment]
@pietro909 waiting your double check to the last changes
you can marge if it's good also for you

**Comment by CrisTofani** [comment]
@pietro909 @Undermaken may we merge this?

**Review by pietro909 — APPROVED** [review]

**Pull request merged**